db/migrate/20230814105634_move_hw_info_to_systems_table.rb

```diff
@@ -0,0 +1,20 @@
+class MoveHwInfoToSystemsTable < ActiveRecord::Migration[6.1]
+  def up
+    safety_assured do
+      execute "update systems as s inner join hw_infos hw on s.id=hw.system_id \
+                set system_information = json_object(\
```

Comment:
In SCC, the JSONB field is called "properties", but in RMT it's called "system_information"? Why the discrepancy?

Reply:
I don't know why that was decided, perhaps @felixsch might be able to answer

Reply:
This is due to the fact that this column **only** ever stores system related information. It is not a generic storage column as in SCC therefore a qualified name suits better then a generic one.

Reply:
I see, thanks!